src/app/configuration/configuration.component.html

```diff
@@ -0,0 +1,98 @@
+<mat-horizontal-stepper [linear]="true" #stepper>
+  <mat-step [stepControl]="loginForm">
+    <ng-template matStepLabel i18n>Create Admin</ng-template>
+    <form [formGroup]="loginForm">
+      <mat-grid-list cols="1">
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Username" formControlName="username" required>
+        </mat-form-field>
+      </mat-grid-list>
+      <mat-grid-list cols="1">
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Password" formControlName="password" type="password" required>
+        </mat-form-field>
+      </mat-grid-list>
+      <mat-grid-list cols="1">
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Confirm Password" type="password" formControlName="confirmPassword" required>
+        </mat-form-field>
+      </mat-grid-list>
+      <button mat-raised-button color="primary" (click)="checkPassword()" i18n>Next</button>
```

Comment:
Instead of checking password matching section in the time of button click, can we do while input value changes ?